app/validators/date_range_validator.rb

```diff
@@ -1,7 +1,17 @@
 class DateRangeValidator < ActiveModel::EachValidator
+  def initialize(**kwargs)
```

Comment:
Please can you add tests for the new validators.  If you have never done a test for a validator before, give me a shout - I can show you an example that I have just done